src/main/java/calculator/StringAdder.java

```diff
@@ -1,10 +1,45 @@
 package calculator;
 
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 /**
  * Created by yusik on 2019/11/02.
  */
 public class StringAdder {
+
+    public static String REGEX_DEFAULT_DELIMITER = "[,:]";
+    public static String REGEX_CUSTOM_DELIMITER = "//(.)\n(.*)";
+
     public int sum(String input) {
-        return 0;
+        if (input.isEmpty()) {
+            return 0;
+        }
+
+        String[] tokens = split(input);
+        int sum = 0;
+        for (String token : tokens) {
+            sum += parseIntOnlyPositive(token);
+        }
+
+        return sum;
+    }
+
+    private int parseIntOnlyPositive(String token) {
+        int number = Integer.parseInt(token);
```

Comment:
```suggestion
        int number = Integer.parseUnsignedInt(token);
```
기존에 제공하고 있는 API를 활용하는 것은 어떤가요.
내부적으로 음수일 경우에 Exception을 던지고 있습니다.

```java
            if (firstChar == '-') {
                throw new
                    NumberFormatException(String.format("Illegal leading minus sign " +
                                                       "on unsigned string %s.", s));
```